app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -202,4 +161,45 @@ protected void onSaveInstanceState(Bundle outState) {
         outState.putCharSequenceArray("savedStates", values);
         outState.putBoolean("server",personalServer.isChecked());
     }
+
+    public void alertSuccess(Response<ForgotPasswordResponse> response) {
+        DialogInterface.OnClickListener dialogClickListener = new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int id) {
+                Intent intent = new Intent(ForgotPasswordActivity.this, LoginActivity.class);
+                intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
+                startActivity(intent);
+                finish();
+            }
+        };
+        alertTitle = null;
+        alertMessage = response.body().getMessage();
+        AlertboxHelper successAlertboxHelper = new AlertboxHelper(ForgotPasswordActivity.this,alertTitle,alertMessage,dialogClickListener,null,"Continue",null,Color.BLUE);
+        successAlertboxHelper.showAlertBox();
+    }
+
+    public void alertFailure() {
```

Comment:
Just combine these functions alertFailure(),alertError(),alertException(). Single method can replace these 3. Pass 'title' and 'message' as parameters. Forgot about second point for now. Will see that in future.